src/main/java/com/openmrs/migrator/MigratorApplication.java

```diff
@@ -16,6 +16,8 @@
 
   private final PDIService pdiService;
 
+  private String[] jobs = {"pdiresources/jobs/merge-patient-job.kjb"};
```

Comment:
@cfaife Lets move this to Configuration Service class so that when someone gets to ticket 100, they only have to focus on settings contained in one file

Reply:
Another thing, for now I don't think we need to plan for the app to run multiple jobs. If we have encounter a use case when we need this later on, we can always add it fairly easily